scripts/migrate-unknown.md

```diff
@@ -0,0 +1,39 @@
+# Migrate Unknown Patients into Restricted Patients
```

Comment:
Not sure when the real migration will take place so this is to remind me what I've been doing. Age is a terrible thing.